item.rb

```diff
@@ -0,0 +1,44 @@
+require 'date'
+# date class for applying archive
+
+class Item
+  attr_accessor :title, :publish_date
+  attr_reader :id, :genre, :author, :label, :archived
```

Comment:
- [ ] Please add getters (attr_reader) for `@source`.

Reply:
![image](https://user-images.githubusercontent.com/13853450/189466073-e5554940-860f-40b9-947b-27ed21f53dc1.png)
consider it done.